SuperteamDAO/earn
# new footer + copy svg

## What does this PR do?
- Added copy svg and brand assets
- added new footer 

## Where should the reviewer start?
- test if copy svg works by pasting in Figma
- download the assets
- check all footer links

## How should this be manually tested?

## Any background context you want to provide?

## What are the relevant issues?

## Screenshots (if appropriate)
